src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx

```diff
@@ -352,7 +375,359 @@ The `amplify-authenticator` component offers a simple way to add authentication
+```
+
+or you can update your `pubspec.yaml` file with the following 
+
+```yaml
+environment:
+  sdk: ">=2.18.0 <4.0.0"
+  flutter: ">=3.3.0"
+
+dependencies:
+  amplify_flutter: ^1.0.0
+  amplify_auth_cognito: ^1.0.0
+  amplify_authenticator: ^1.0.0
+```
+
+and run the following command to download the libraries.
+
+```sh showLineNumbers={false}
+flutter pub get
+```
+
+Next, update your `main.dart` file with the following:
+
+```dart
+import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
+import 'package:amplify_authenticator/amplify_authenticator.dart';
+import 'package:amplify_flutter/amplify_flutter.dart';
+import 'package:flutter/material.dart';
+
+import 'amplifyconfiguration.dart';
 
+Future<void> main() async {
+  try {
+    WidgetsFlutterBinding.ensureInitialized();
+    await _configureAmplify();
+    runApp(const MyApp());
+  } on AmplifyException catch (e) {
+    runApp(Text("Error configuring Amplify: ${e.message}"));
+  }
+}
+
+// highlight-start
+Future<void> _configureAmplify() async {
+  try {
+    await Amplify.addPlugin(AmplifyAuthCognito());
+    await Amplify.configure(amplifyConfig);
+    safePrint('Successfully configured');
+  } on Exception catch (e) {
+    safePrint('Error configuring Amplify: $e');
+  }
+}
+// highlight-end
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+  @override
+  Widget build(BuildContext context) {
+    // highlight-next-line
+    return Authenticator(
+      child: MaterialApp(
+        // highlight-next-line
+        builder: Authenticator.builder(),
+        home: const Scaffold(
+          body: Center(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                // highlight-next-line
+                SignOutButton(),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+```
+</InlineFilter>
+
+<InlineFilter filters={['swift']}>
+
+Move the generated files to your project. You can do this by dragging and dropping the files to your project.
+
+![Shows the drag and drop phase](/images/lib/getting-started/ios/set-up-swift-8.png)
+
+Open your project in Xcode and select File > Add Packages... and add the following dependencies:
+
+- Amplify Library for Swift: Enter its (GitHub URL)[https://github.com/aws-amplify/amplify-swift], *select Up to Next Major Version* and click *Add Package*
+
+  - Select the following libraries:
+    - Amplify
+    - AWSCognitoAuthPlugin
+
+- Amplify UI Swift - Authenticator: Enter its (GitHub URL)[https://github.com/aws-amplify/amplify-ui-swift-authenticator], *select Up to Next Major Version* and click *Add Package*
+
+  - Select the following library:
+    - Authenticator
+
+Once you are done, add the API dependencies to your project. Select **File > Add Package Dependencies...** and add the `AWSAPIPlugin`.
+
+![Shows the Amplify API library for Swift selected](/images/lib/getting-started/ios/set-up-swift-9.png)
```

Comment:
Could we get a more descriptive alt text for this to better help users who are using screen readers?

Reply:
Done ✅